core/templates/dev/head/pages/teach.html

```diff
@@ -48,56 +48,51 @@
-              <p style="margin-top: -10px;">
-                to improve your exploration.
-              </p>
-            </div>
+          <div style="text-align: center; width: 100%;">
+            <em>Create high-quality lessons that help students learn difficult topics.</em>
           </div>
 
-          <hr>
+          <p>
+            You might know a student who's struggling with a topic at school.
+            They really want to learn it, but the lesson goes too fast, or the
+            textbook isn't easy to understand &mdash; and it's hard for them to
+            get the support they need. And it's not just this student: other
+            students, all over the world, are likely to struggle with the same
+            topics, again and again and again and again.
+          </p>
 
           <p>
-            How do pendulums work? Is light a particle or a wave? What is art?
-            Can you explain topics like these to a student who wants to learn
-            more? If so, you can use Oppia's unique technology to recreate 1-on-1
-            tutoring and reach students around the world by creating and sharing
-            a single piece of learning material.
+            In order to help address this, we're excited to announce the
+            <em>Teach with Oppia</em> program!
           </p>
 
           <p>
-            With Oppia, you can create <em>explorations</em> that help students
-            learn in a fun and creative way, using videos, images, and open-ended
-            questions. Students often have similar misconceptions, and Oppia
-            helps you organize these so that you can give targeted feedback to a
-            group of students at a time. By teaching with Oppia, you can
-            communicate with students around the world, understand different
-            points of view, and improve your skills in communication and empathy.
+            The <em>Teach with Oppia</em> program enables <strong>anyone</strong>
+            &mdash; especially students &mdash; to contribute to a collection
+            of free, accessible lessons on topics that learners find difficult.
+            The lessons use Oppia's unique technology to recreate 1-on-1
+            tutoring, and typically include clear, compelling and motivated
+            explanations, together with ample exercises that allow students to
+            gain proficiency with the subject. Students can then do these
+            lessons at their own pace, supported as much as possible by useful,
+            targeted feedback.
           </p>
+
+          <p>
+            Currently, participation in the program is <strong>free</strong>.
+            Participants will receive personalized feedback on the lessons they
+            create, as well as help with playtesting and marketing their
+            lessons. They'll also be able to connect with other like-minded
+            creators, as well as the Oppia development team if additional
```

Comment:
Good catch, though I can't think of another way to reword it -- "and" is overused, too.